src/screenshot.h

```diff
@@ -36,7 +36,8 @@ public slots:
     void topWindowScreenshot();
     void noNotifyScreenshot();
     void savePathScreenshot(const QString &path);
-
+    void multioptionalScreenshot(int delay, int areaOption, QString savePath);
+    void multioptionalScreenshot(int areaOption, QString savePath);
```

Comment:
we'd better use a function name like `screenshotWithOptions`?

Reply:
Thanks for your suggestions, I'd improve and update this branch later ☺